config/default/tox-envlist.j2

```diff
@@ -1,6 +1,7 @@
 [tox]
 minversion = 3.18
 envlist =
+    release-check
```

Comment:
I think this env should also run on GHA but it should be allowed to fail without failing the build.
So the release manager is not surprised when running this env before the release that fixes are needed. (I personally rely on GHA being green before the release, I do not always run the tests localy before cutting a release.)